frontend/src/widgets/TotalHrsAndRecipientGraph.scss

```diff
@@ -88,6 +88,12 @@
   z-index: 1;
 }
 
+@media(max-width: 1606px) {
+  .ttahub-legend-control {
+    margin-top: .5rem;
+    margin-bottom: .5rem;
+  }
```

Comment:
not using the USWDS utility classes since they don't have a breakpoint that matches the behavior we want